flow-typed/npm/array-range_vx.x.x.js

```diff
@@ -1,33 +0,0 @@
-// flow-typed signature: 64b04cdeef33666b68764088f3e8c439
```

Comment:
This may be controversial and I'll remove if you disagree with this. I don't see any reason why we should even have these around, as they just add to the noise of our type defs. I also added a command to automate the removal for us.